LAD-AMA-Common/metrics_ext_utils/metrics_ext_handler.py

```diff
@@ -338,9 +338,9 @@ def start_metrics(is_lad):
         az_resource_id, subscription_id, location, data = get_imds_values(is_lad)
 
         if is_lad:
-            monitoringAccount = "CUSTOMMETRIC_"+ subscription_id
```

Comment:
I assume this does not introduce any possible back compat issues where older version of ME might still be on the machine expecting upper case?

Reply:
I am on an email thread with Michael. I'll ask him this to confirm as well. But we have to include the latest ME in the next release anyways due to the Custom metrics folk's ask (Not sure if that's only for LAD or for AMA as well, Luke will know).  So, this change should only go when we update the ME we ship with our extensions